pkg/repo/validate.go

```diff
@@ -32,6 +32,8 @@ func (r *Repo) Validate() (
 	valErrMap map[string][]error,
 	err error,
 ) {
+	valErrMap = map[string][]error{}
```

Comment:
This not being initialized was causing the error map to not get populated below.

Reply:
FYI https://github.com/kubernetes/enhancements/pull/2676
@saschagrunert